README.md

```diff
@@ -195,4 +195,4 @@ operator-sdk cleanup packagemanifests --olm-namespace openshift-operator-lifecyc
 ```
 
 *Operator-sdk has a known bug while using `operator-sdk run/cleanup --olm` where it shows failure on success. 
-Track the issue [here](https://github.com/operator-framework/operator-sdk/issues/2938). The error does not imply that the operator will not work.*
\ No newline at end of file
+Track the issue [here](https://github.com/operator-framework/operator-sdk/issues/2938). The error does not imply that the operator will not work.*
```

Comment:
What is the change here?

Reply:
I didn't understand it either.

Reply:
I know the README is missing a newline at the end. Maybe you added it though that would be shown differently. Did you make this change from a Mac or Windows machine?

Reply:
I am on my linux laptop :(

Reply:
OK, please add a new line and let us get this merged.

Reply:
Reminder to add a new line.

Reply:
It is there already. I pushed last week but the UI is confusing as it won't show new line that.